backend/lib/watches/journals.js

```diff
@@ -66,5 +69,24 @@ module.exports = (io, retryOptions = {}) => {
     }
   }
 
-  return loadAllOpenIssues()
+  let pollIntervalSeconds = parseInt(config.gitHub.pollIntervalSeconds)
+  if (isNaN(pollIntervalSeconds)) {
+    pollIntervalSeconds = undefined
+  }
+  return pForever(async () => {
+    await loadAllOpenIssues()
+    if (!pollIntervalSeconds) {
+      return pForever.end
+    }
+
+    const issueNumbers = journalCache.getIssueNumbers()
+    await _.forEach(issueNumbers, async number => {
```

Comment:
does not work as expected. Use normal for loop
```js
for (const number of issueNumbers) {
  try {
    await journals.loadIssueComments({ number })
  } catch (err) {
    logger.error('failed to fetch comments for reopened issue %s: %s', number, err)
  }
}
```

Reply:
✅